typescript/lib/parsers/palm.ts

```diff
@@ -232,7 +245,7 @@ function constructOutputs(
     const candidate = response.candidates[i];
     const output: ExecuteResult = {
       output_type: "execute_result",
-      data: candidate.output,
+      data: candidate.output ?? "",
```

Comment:
Maybe `?? null`? See: #631 as well

Reply:
Our data object must be a string. This is palm which is purely text so it cannot be a function call or any other data type